lib/safari-samesite.js

```diff
@@ -0,0 +1,73 @@
+/*!
+ * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+ */
+import bedrock from 'bedrock';
+import 'bedrock-express';
+import onHeaders from 'on-headers';
+
+bedrock.events.on('bedrock-express.init', app => {
+  app.use((req, res, next) => {
+    safariSameSiteFix(req, res, next);
+  });
+});
+
+export function safariSameSiteFix(req, res, next) {
+  // const headers = res.header();
+  // console.log(headers);
+  onHeaders(res, function() {
+    /* Note: This fix is for Safari browsers before version 14.1.x. Those
+    browsers did not handle the cookie attribute and value `SameSite=None`.
+    They either treated it as `SameSite=Strict` or broke in other ways. The
+    fix is to not set that attribute for those browsers. That fix is
+    implemented here. */
+    const setCookie = res.getHeader('set-cookie');
+    if(!setCookie) {
+      return;
+    }
+    const userAgent = req.headers['user-agent'];
```

Comment:
```suggestion
    const userAgent = req.get('user-agent');
```

Reply:
I can't do this because the tests fail with this implementation. `req.get` is not a function if I change to this for the tests.

Reply:
This has been update now due to the new test suite changes.